__tests__/__fixtures__/docs/existing-docs/subdir/another-doc.md

```diff
@@ -1,7 +1,7 @@
 ---
 title: This is another document title
 category:
-  uri: /branches/stable/categories/guides/category-slug
+  uri: category-slug
```

Comment:
Updating so we can use it with `references upload` as well

Reply:
update: worked on this in https://github.com/readmeio/rdme/pull/1262!